### DISCUSSION TITLE
Self-hosted kits

### DISCUSSION THREAD

**Discussion by LeaTark** [FortAwesome/Font-Awesome#20953] [Feature Requests]
Would there be any chance of something like self-hosted kits?

e.g. one that uses the local individual .svg files instead of sending requests to FontAwesome’s servers and doesn’t have the bloat of all.js

I think that would hit the best-of-both-worlds sweet spot.

**Comment by wzwizu** [comment]
Why is this from 2020 and still not even has one response? This should already be a feature right? I mean now the FA cdn is a dependency for all kit based projects....

**Comment by ArniPL** [comment]
I was just searching for this, and I think this is something worth exploring.

**Comment by halloei** [comment]
Looks like the feature has landed: https://blog.fontawesome.com/downloadable-kits-are-here/

**Comment by tagliala** [comment]
Thanks for the heads-up!

**Comment by christhofer** [comment]
> Looks like the feature has landed: https://blog.fontawesome.com/downloadable-kits-are-here/

This is the [documentation](https://fontawesome.com/docs/web/setup/host-yourself/svg-js#hosting-your-own-downloaded-kit) to use the self-host kit.
Turns out we still need to import solid.js, regular.js and so on which are big and wasteful.
```html
<script defer src="/your-path-to-fontawesome/js/solid.min.js"></script>
<script defer src="/your-path-to-fontawesome/js/regular.min.js"></script>
<script defer src="/your-path-to-fontawesome/js/brands.min.js"></script>
<script defer src="/your-path-to-fontawesome/js/fontawesome.js"></script>
```

Unlike the online kits that just read the HTML and download the required SVG files.
```html
<script src="https://kit.fontawesome.com/abcdefg12345.js" crossorigin="anonymous"></script>
```

Here is the screenshot of the docs as of writing this comment.
 ![Screenshot in case the docs gets updated later](https://github.com/FortAwesome/Font-Awesome/assets/13524958/116bbaa7-84da-4aa0-be9e-64c6c667921c)

**Comment by ArniPL** [comment]
So the issue is still not resolved. @tagliala could we reopen, please?
